gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Jan 12 13:39:55 MSK 2016
+#Fri May 06 11:55:15 EDT 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-all.zip
```

Comment:
I updated to Android Studio 2.+ before taking on this project who had an older Android Studio version. I had to perform a series of updates and syncs including gradle version in order to achieve compatibility and this line must have been one of the changes it made.